### ISSUE TITLE
Wizard migration - UX deliverables for education flow

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#13374]
**Acceptance Criteria**
- Create hi-fi mock-ups and documentation based on Mikki's [education wizard flow](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1599137761089/b5b2662802d862b436660938241e79d1ef4ecf31?wid=0-1599583230985)

**Outstanding questions**
- Is our proposed change to the wizard behavior feasible? Wizard is skipped only when sent to new form after completing it.  Once you navigate away, you will need to repeat the wizard. 
- Should all the EDU wizards have a skip link ('no thanks, i'm confident this is the right form')?
- What is the difference in content needed across all of the education forms?
- Confirm the Wizard should NOT be on the following Apply Now pages (to be confirmed):
  - 0994 - Apply for VET Tech
  - 1990n - Apply for Education Benefits under the National call to service program
  - Form 22-10203 - Apply for Rogers STEM Scholarship
  - 5495 - Dependents' Request for Change of Program or Place of Training
  - 1990e - Apply to use transferred education benefits
  - 5490 - Apply for Education Benefits as an Eligible Dependent

**Tasks**
- [x] Evaluate whether or not we need to create hi-fis for each form, or if one will do.
- [x] Create hi-fi for `/introduction` page(s)
   - include skip link where appropriate
   - include examples showing messaging indicated which form the user will be directed to
   - Reflect any updates made to h1s and breadcrumbs for each form 
- [x] Create hi-fi for subsequent view when user is on the correct form, showing subway map
   - Add content/link to "start over"
   - Provide guidance/visuals for interaction going between views
- [x] Create hi-fi for subsequent view when user is directed to a new form, showing subway map
   - Add content/link to "start over"
   - Provide guidance/visuals for interaction going between views

**Comment by MickinSahni** [comment]
not committing to this for sprint 31 until we're in the clear with existing sprint priorites and the blocker (#13379)

**Comment by andaleliz** [comment]
@peggygannon @mnorthuis @johnhashva, I put together some screens for the education flow. The [UX pin prototype](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages//simulate/no-panels?mode=i) includes:
- the [1990 wizard page](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages//simulate/no-panels?mode=i), when a user answers the wizard questions in such a way that the 1990 is the correct form for them. There is a skip link on this page
- The [logged out subway map/intro page for the 1990](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages/132982271/simulate/no-panels?mode=i), with options to go back.
- The [logged in subway map/intro page for the 1990](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages/132983474/simulate/no-panels?mode=i), with options to go back.  This work really shouldn't impact anything differently on the authenticated side, but created this because I wanted to explore and double check that all it really needs is the `go back` links

I think the UX will be identical across **all** the education forms, and that it makes sense to get content before I create screens for any other scenarios.  If the UX is indeed the same for all the EDU forms, we should discuss what makes the most sense to deliver to teams in terms of visuals and documentation. I want to provide all the artifacts needed to support them without being unnecessarily redundant.

There's a [screen in the prototype showing a SIP application scenario](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages/132983475/simulate/no-panels?mode=i) too. I asked Mikki about this on another ticket and captured it in the prototype to make sure we don't miss this use case.

**Comment by andaleliz** [comment]
[The prototype](https://preview.uxpin.com/ca08f3dbf63475b2b62f3e8c00050ad1b19c4cb5#/pages//simulate/no-panels?mode=i) has been updated with most of the alerts. I need a few of things from @peggygannon  before I can complete all the mock-ups:

- Can we incorporate the STEM eligibility info into the wizard flow instead of the current bulleted list that is there?
- Copy for links under the CTA buttons in the Edu alerts for each flow
- Condensed wording for the `Are you receiving education benefits transferred to you by a sponsor Veteran?` question

**Comment by peggygannon** [comment]
@andaleliz Updated EDU outline is here: 
[Education wizard alerts.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5337377/Education.wizard.alerts.docx)

and answers to your questions above: 

Can we incorporate the STEM eligibility info into the wizard flow instead of the current bulleted list that is there?

- I discussed this with Matt and Amy and we don't want to go the route because the user has to answer the questions again while they're in the form itself and we didn't want to force them to answer the same set of questions twice.

- Also, we want to keep the eligibility requirements in the wizard so potential applicants have the greatest chance of actually seeing the eligibility criteria.

Copy for links under the CTA buttons in the Edu alerts for each flow

- I've included this in above outline, as well as updated link to the Edith Nourse Scholarship page  to point to the VA.gov page 

Condensed wording for the Are you receiving education benefits transferred to you by a sponsor Veteran? question
- this is in outline above!

**Comment by andaleliz** [comment]
Thanks @peggygannon! Makes sense about the STEM eligibility information.

The prototype has been updated with all of the new content. I realized one more piece we need: the language for the `go back`/`start over` option. It currently says:

If you don't think this is the right form, [go back](return user to wizard they came from)

which probably isn't the right verbiage.

I included a few mobile screens to ensure this all translates smoothly. There are some mobile improvements the design system components on these pages could use, but I didn't reflect them here since that's part of a larger issue.

Once our smaller team reviews these, I _think_ we can hand off to the Edu team for implementation.

**Comment by johnhashva** [comment]
@andaleliz Adding for posterity here some updates I garnered  this week in meetings with Matt Self and Amy Knox (Rogers STEM). Some of these were already posted to Slack:

- With Matt's sign-off (and Amy's agreement on Rogers STEM), we will place the Wizard on **all application** pages (as per original guidance).  This is in keeping with the goal of putting the Wizards in front of users when they apply to help ensure they are applying for the correct benefit -- and are likely eligible.

- Amy wanted us to confirm that the flow for Rogers STEM will be (I confirmed this is the approach -- but validating here):
   - Wizard will be on the Apply for Education Benefits page (/1990/education) and if user answers questions for Rogers STEM s/he will be linked directly to the Form Application Intro page (https://www.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction)
  - If user navigates (via search, marketing, etc.) to the Rogers STEM "landing page'(https://www.va.gov/education/other-va-education-benefits/stem-scholarship/), s/he will see the Wizard.

**Comment by andaleliz** [comment]
@johnhashva capturing what we discussed in our sync today here, wrt to Amy's concerns:
>Wizard will be on the Apply for Education Benefits page (/1990/introduction) and if user answers questions for Rogers STEM s/he will be linked directly to the Form Application Intro page (https://www.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction)

Yes, that's correct

>If user navigates (via search, marketing, etc.) to the Rogers STEM "landing page'(https://www.va.gov/education/other-va-education-benefits/stem-scholarship/), s/he will see the Wizard.

No, the wizard won't be on that URL; it would be on the URL people are sent to when they click the `apply` button on the page you referenced, which is https://www.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction

**Comment by andaleliz** [comment]
2 more items outstanding before we can close this out:
1. text for `start over` link
2. confirmation whether or not we will change `h1` elements on our pages.

**Comment by peggygannon** [comment]
hi, @andaleliz here is the Go back link language for the subway map screens: 

If you don’t think this is the right form for you, [go back and answer questions again](return user to wizard they came from)

**Comment by andaleliz** [comment]
@peggygannon thank you - updates have been pushed to the prototype.

**Comment by andaleliz** [comment]
`h1` elements on intro pages have been updated per Peggy's doc via Slack.  Closing this out!
